pkg/volume/util/operationexecutor/operation_generator.go

```diff
@@ -596,6 +596,7 @@ func (og *operationGenerator) GenerateMountVolumeFunc(
 			}
 
 			resizeOptions.DeviceMountPath = deviceMountPath
+			resizeOptions.DeviceStagePath = deviceMountPath
```

Comment:
Why are stage and mount path the same here

Reply:
The reason `DeviceStagePath`  was introduced is to make it easier for drivers to find out the staging path if they support node-staging. So stage and mount path are always going to be same for fstype volumes. The only difference is, `DeviceStagePath` will not even be set for volume types that do not support node-staging.

Reply:
Can you add a comment describing this? It's very confusing. We call node expand after stage and again after publish. And the resize call after stage will have the same paths, but the one after publish will have different paths?

Reply:
Added a comment explaining this and also marked this code flow as deprecated because in future we will not be calling `NodeExpandVolume` in between stage and publish.